flag.go

```diff
@@ -96,6 +96,10 @@ type DocGenerationFlag interface {
 	// GetValue returns the flags value as string representation and an empty
 	// string if the flag takes no value at all.
 	GetValue() string
+
+	// GetTakesFile indicates that the command takes a file as input, which
+	// will be used for documentation and completion generation purposes.
+	GetTakesFile() bool
```

Comment:
Given we have TakesValue() this should be TakesFile()?

Reply:
Golang will complain if we have a method and a value with the same name and it seems that we follow a `Get...` pattern for accessing the values from the interface. WDYT?

Reply:
Then why do we a method at all?

Reply:
I see, I removed it completely.